src/ui/rideWindow.ts

```diff
@@ -115,6 +116,28 @@ export const rideWindow = window({
 				})
 			]
 		}),
+		groupbox({
+			text: "Ride name",
+			tooltip: "Edit the name of the ride",
+			content: [
+				label({
+					text: model._title,
+					height: 14
+				}),
+				button({
+					text: "Rename ride",
+					height: buttonSize,
+					onClick: () => {
+						ui.showTextInput({
+							title: "Ride/attraction name",
+							initialValue: model._title.get(),
+							description: "Enter new name for this ride/attraction:",							
+							callback: (input) => context.executeAction("ridesetname", {ride: model._ride.get()?._id, name: input})
```

Comment:
Shouldn't we update the store as well once the `ridesetname` action is successfully completed?

Reply:
added it.